backup-unit/browserunit.cpp

```diff
@@ -0,0 +1,105 @@
+#include <vault/unit.h>
+#include <qtaround/os.hpp>
+
+#include <QProcess>
+#include <QCoreApplication>
+#include <QStringList>
+#include <QVariantList>
+#include <QVariantMap>
+#include <QDebug>
+#include <sys/types.h>
+#include <signal.h>
+#include <set>
+#include <unistd.h>
+#include <stdexcept>
+
+
+void stop_browser()
+{
+    qDebug() << "Terminating browser";
```

Comment:
You could introduce:

```
Q_LOGGING_CATEGORY(lcCoreLog, "org.sailfishos.browser.backup", QtWarningMsg)
```
to the logging.cpp and declare it in logging.h. As this is standalone binary we could compile those into this project directly.

When above is added then qCDebug(lcBackupLog) for qDebug and qCWarning for qWarnings.